src/test/javascript/spec/component/team/teams-import-from-file-form.component.spec.ts

```diff
@@ -134,4 +135,19 @@ describe('TeamsImportFromFileFormComponent', () => {
             expect(comp.convertTeams(mockFileStudents)).to.deep.equal(mockFileTeamsConverted);
         });
     });
+
+    describe('Invalid team name throws exception', () => {
+        beforeEach(() => {
+            resetComponent();
```

Comment:
Before each for one test can be shortened by moving the `resetComponent();` into the _should throw error_